prow/plank/reconciler.go

```diff
@@ -95,6 +95,9 @@ func add(
 	if err := mgr.GetFieldIndexer().IndexField(ctx, &prowv1.ProwJob{}, prowJobIndexName, prowJobIndexer(cfg().ProwJobNamespace)); err != nil {
 		return fmt.Errorf("failed to add indexer: %w", err)
 	}
+	if err := mgr.GetFieldIndexer().IndexField(ctx, &prowv1.ProwJob{}, prowJobQueueNameIndexName, prowJobQueueNameIndexer(cfg().ProwJobNamespace)); err != nil {
```

Comment:
Please augment the existing `prowJobIndexer` rather than defining another.

Reply:
@cjwagner this adds a new index, using the existing index is possible but requires some kind of prefix to avoid collisions (as you mentioned inside the indexer func) - What drawbacks do you see with adding a new index?

Reply:
I didn't realize that a separate index name was being used as an alternative to using a value prefix, that makes more sense. I was also thinking of using a prefix for consistency with the existing indexing logic and because I wasn't sure about the performance cost of additional indexers. Both these concerns are negligible though.

Reply:
I believe that reusing the existing indexer is indeed a cleaner approach so I changed the code to do just that. Fixed in https://github.com/kubernetes/test-infra/pull/25674/commits/55d9a8ef93e1eccd103b77a00f96d7a84cf0cbbd.